src/stores/commandStateStore.ts

```diff
@@ -36,7 +36,7 @@ export const updateCommandState = () => {
   const action =
     selection.isActive() && selection.isOnThought()
       ? getCommandState(selection.html() ?? '')
-      : getCommandState(pathToThought(state, state.cursor).value)
+      : getCommandState(pathToThought(state, state.cursor)?.value ?? '')
```

Comment:
on that end we have a `path.length === 0` default case, so fallback is fine here